spring-cloud-dataflow-scheduler-launcher/src/main/java/org/springframework/cloud/dataflow/scheduler/launcher/configuration/SchedulerTaskLauncher.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.scheduler.launcher.configuration;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.StreamSupport;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import org.springframework.cloud.dataflow.rest.client.DataFlowClientException;
+import org.springframework.cloud.dataflow.rest.client.TaskOperations;
+import org.springframework.cloud.dataflow.rest.resource.LauncherResource;
+import org.springframework.core.env.AbstractEnvironment;
+import org.springframework.core.env.EnumerablePropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.core.env.MutablePropertySources;
+import org.springframework.hateoas.PagedModel;
+import org.springframework.util.Assert;
+import org.springframework.util.StringUtils;
+
+/**
+ * Sends a request to the Spring Cloud Data Flow server to launch a task.
+ *
+ * @author Glenn Renfro
+ **/
+public class SchedulerTaskLauncher {
+
+	static final String TASK_PLATFORM_NAME = "spring.cloud.dataflow.task.platformName";
```

Comment:
We already have this property in a few other places and hence, can we move this constant to a common accessible helper class?

Reply:
Hmm...   🤔 
I like the idea but this is used in 3 different modules 
a) Task Command 
b) SchedulerTask Launcher
c) And DefaultTaskExecutionService.

While we could technically add it to a helper class  in the  spring-cloud-dataflow-server-core and that would work for SchedulerTaskLauncher and DefaultTaskExecutionService.  However I don't feel comfortable adding that dependency to the module containing TaskCommand.  
Do you know of a good place to put the helper class?